<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="" />


<title>Relationship between Evolutionary Divergence and Historical Connectivity in the Northern Andean Páramos During the Pleistocene: The Case of Espeletiinae</title>

<script src="site_libs/header-attrs-2.14/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>










<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}

.tocify-subheader {
  display: inline;
}
.tocify-subheader .tocify-item {
  font-size: 0.95em;
}

</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html"></a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Relationship between Evolutionary
Divergence and Historical Connectivity in the Northern Andean Páramos
During the Pleistocene: The Case of Espeletiinae</h1>
<h4 class="author"></h4>

</div>


<style>
/* Mantiene la misma fuente para los blockquotes */
blockquote {
    font-family: inherit; /* Usa la misma fuente que el resto del texto */
    font-size: inherit; /* Usa el mismo tamaño de fuente que el resto del texto */
}

/* Ajusta el ancho del menú de la tabla de contenido flotante */
.tocify {
    width: 1300px; /* Ajusta este valor según sea necesario */
}
.tocify-wrapper {
    width: 1300px; /* Ajusta este valor según sea necesario */
}
</style>
<hr />
<div id="introduction" class="section level1">
<h1><strong>1. Introduction</strong></h1>
<p>This study provides a recent historical perspective aimed at better
understanding the diversification processes of organisms in the northern
Andean páramos, a high-mountain ecosystem that has been significantly
influenced by geological and climatic events during the Pleistocene.
During this time, climatic fluctuations generated complex dynamics of
connectivity among the páramo fragments, which are hypothesized to have
affected the distribution and divergence of species inhabiting these
ecosystems. The objective of this study is to evaluate whether there is
a relationship between evolutionary divergence and historical
connectivity during the Pleistocene, aiming to identify which aspects of
this connectivity are important in the speciation process. To achieve
this, we integrate paleoclimatic modeling of the páramo boundaries over
the last 2 million years with the evolutionary history of a complex of
frailejones (subtribe Espeletiinae) as a biological model.</p>
<hr />
<p><strong>Note</strong>: <em>The titles of key methodological sections
include links that direct to GitHub repositories with the codes used and
more detailed explanations for each section. To review the complete
process of this study, excluding the phylogenetic reconstruction, visit
the following <a
href="https://innerhaze.github.io/Evolutionary-Divergence-Historical-Connectivity-in-Northern-Andean-Paramos-Pleistocene-/">link.</a></em></p>
<hr />
<div id="context" class="section level3">
<h3><strong>1.2 Context</strong></h3>
<p>Three key processes shape divergence during speciation: gene flow,
selection, and genetic drift. Gene flow and drift are profoundly
influenced by the spatial configuration of the habitat, while selection
depends on the local biotic and abiotic environment. These processes are
believed to have a more pronounced impact in the early stages of
speciation, before prezygotic barriers are established, which interrupt
gene flow between lineages. Having a habitat distribution model spanning
a considerable time interval where speciation events can occur provides
an ideal opportunity to test this theoretical framework.</p>
<p>Frailejones are plants closely tied to the climate, occurring
exclusively at high elevations where arboreal plants are absent. Their
evolution has been shaped by the spatial dynamics of the páramo during
the Pleistocene, a period during which many of their current species
originated.</p>
<p>This study represents a first attempt to quantify the effect of
habitat spatial configuration over time, evaluating how historical
connectivity and area size have influenced gene flow and genetic drift
during the speciation process.</p>
<p>What time intervals and conditions are crucial for speciation? This
is a fascinating question that has captured the interest of many
researchers. However, the resources needed to address this question are
limited. Therefore, this scenario provides an excellent opportunity to
examine how spatial configuration over time can shed light on
diversification processes.</p>
<hr />
</div>
</div>
<div id="methodology" class="section level1">
<h1><strong>2. Methodology</strong></h1>
<hr />
<div id="geographic-connectivity-models" class="section level3">
<h3><strong>2.1 Geographic Connectivity Models</strong></h3>
<p>The study began with the development of geographic connectivity
models, based on the paleoclimatic model proposed by Flantua et
al. (2019). This model describes the fluctuations of the Upper Forest
Line (UFL) over ≈2 million years, allowing us to assess how these
changes affected connectivity between the páramo fragments in the
Andes.</p>
<ol style="list-style-type: decimal">
<li><div>

<blockquote>
<p><a
href="https://github.com/innerhaze/DEM-Reclassification-R-Paramo-Connectivity-Modeling"><strong>Development
of the Movement Cost Matrix:</strong></a> A cost raster was constructed
using a Digital Elevation Model (DEM) encompassing all páramo areas in
the northern Andes. Specific movement costs were assigned to different
altitudinal zones, reflecting the limited dispersal capacity of páramo
organisms. The movement cost gradually increases as one moves away from
the páramo core, reflecting the difficulties faced by species when
dispersing between isolated fragments.</p>
</blockquote>
<div>
</li>
<li><div>

<blockquote>
<p><a
href="https://github.com/innerhaze/Java-R-Integration-with-Graphab-Cost-Surface-Processing-and-Data-Management"><strong>Calculation
of Connectivity Metrics:</strong></a> Using the Graphab software,
various connectivity metrics were calculated, including Least-Cost
Distance (LCD) and the Current Flow (CF) index. These metrics were
determined for 19 distinct páramo configurations, derived from the
Pleistocene climatic fluctuations according to the Flantua et al. (2019)
model. These calculations are crucial for understanding how climatic
fluctuations influenced connectivity between the patches representing
the distribution of each species included in the Espeletiinae phylogeny
developed by Diazgranados and Barber (2017).</p>
</blockquote>
<div>
</li>
</ol>
<hr />
</div>
<div id="phylogenetic-methods" class="section level3">
<h3><strong>2.2 Phylogenetic Methods</strong></h3>
<p>With the connectivity models established, we proceeded to construct
an ultrametric phylogeny of Espeletiinae. DNA sequence data from the
ETS, ITS, and rpl16 genes, obtained from previous studies by
Diazgranados and Barber (2017), were used. These sequences were
concatenated, and the IQ-TREE software was used to build the phylogeny
based on the GTR+I+G model. The tree was calibrated considering key
historical events, such as the divergence between Espeletiinae clades in
Venezuela and Colombia, placing the crown of the tree at approximately
2.3 million years, following the methodology of Pouchon et
al. (2021).</p>
<hr />
</div>
<div id="historical-connectivity-algorithm" class="section level3">
<h3><strong>2.3 Historical Connectivity Algorithm</strong></h3>
<p>An algorithm developed in R was used to integrate the information
from the connectivity models and the phylogeny. This algorithm weights
connectivity metrics based on the duration of each páramo configuration
during the time interval between parent and child nodes in the
phylogeny. This interval is selected to represent the period during
which, hypothetically, the Pleistocene climatic fluctuations would have
influenced connectivity, increasing or decreasing isolation between
species. Once these key times were established, the algorithm queries
the Flantua et al. (2019) paleoclimatic model database to obtain the
páramo configurations and their duration during these intervals. With
this information, a weighted average is calculated that reflects
historical connectivity for each pair of phylogenetic combinations,
ensuring that the critical divergence periods between species are
appropriately considered in the analysis, provided these times do not
exceed 2.2 million years, the time range covered by the paleoclimatic
model. Visit the following <a
href="https://github.com/innerhaze/R-Algorithm-for-Historical-Connectivity-Analysis-of-Paramo-Lineages-in-the-Pleistocene">link</a>
for more details.</p>
<hr />
</div>
<div id="generalized-dissimilarity-model-gdm" class="section level3">
<h3><strong>2.4 Generalized Dissimilarity Model (GDM)</strong></h3>
<p>After generating the historical connectivity data from the designed
algorithm, a Generalized Dissimilarity Model (GDM) was used to assess
the relationship between historical connectivity and evolutionary
divergence in Espeletiinae. The GDM allowed us to analyze how the
historical connectivity metrics, calculated based on páramo
configurations during the Pleistocene, correlate with the evolutionary
distances between lineages. Visit the following <a
href="https://github.com/innerhaze/Generalized-Dissimilarity-Modeling-for-Evolutionary-Divergence-and-Historical-Connectivity-Analysis">link</a>
for more details.</p>
<hr />
</div>
</div>
<div id="main-results" class="section level1">
<h1><strong>3. Main Results</strong></h1>
<p>The results indicate that geographic connectivity during the
Pleistocene had a significant impact on the evolutionary divergence of
Espeletiinae. In particular, historical connectivity emerged as a key
explanatory variable for evolutionary distances when the patches
(occurrence sites) were farther apart. In contrast, when the patches
were closer together, Euclidean distance proved to be a more determining
factor.</p>
<hr />
</div>
<div id="methodological-issues-and-challenges" class="section level1">
<h1><strong>4. Methodological Issues and Challenges</strong></h1>
<p>Below are some issues and challenges to address within the
methodological aspects:</p>
<ol style="list-style-type: decimal">
<li><div>

<blockquote>
<p><strong>Temporal occurrence data:</strong> The occurrence of each
lineage in the landscape is based on current data. Therefore,
connectivity calculations are made based on the present distribution, as
no precise information is available on the historical location of these
organisms. This is clearly illustrated in figure 2 of this <a
href="https://innerhaze.github.io/R-Algorithm-for-Historical-Connectivity-Analysis-of-Paramo-Lineages-in-the-Pleistocene/">link</a></p>
</blockquote>
<div>
</li>
<li><div>
<blockquote>
<p><strong>Complexity in measuring connectivity:</strong> When species
occur within the same patch, measuring connectivity in Graphab becomes
complicated. This is because two or more species, when located in
adjacent occurrence points, are treated as a single patch unit,
complicating measurements intended to be made between species pairs.</p>
</blockquote>
</div></li>
<li><div>

<blockquote>
<p><strong>Temporal limitations in phylogenetic studies:</strong>
Phylogenies studied using this method cannot exceed 2.2 million years in
their study times, as this is the maximum data range available in the
pollen column used.</p>
</blockquote>
<div>
</li>
</ol>
<hr />
</div>
<div id="references" class="section level1">
<h1><strong>5. References</strong></h1>
<ol style="list-style-type: decimal">
<li>Diazgranados, M., &amp; Barber, J. C. (2017). The endemic Espeletia
complex: Phylogenetics, morphological evolution, and biogeography of the
most diverse plant clade in the high Andes. Frontiers in Plant Science,
8, 2058.</li>
<li>Pouchon, C., et al. (2021). Phylogenetic signatures of ecological
divergence and leapfrog adaptive radiation in Espeletia. American
Journal of Botany, 108(1), 113-128.</li>
<li>Flantua, S. G. A., O’Dea, A., Onstein, R. E., Hooghiemstra, H.,
&amp; Hoorn, C. (2019). The flickering connectivity system of the north
Andean páramos. Journal of Biogeography, 46(8), 1808-1825.</li>
</ol>
<hr />
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = false;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
